simulator-ui/tests/home.spec.ts

```diff
@@ -0,0 +1,169 @@
+});
+
+//should there be two separate tests for this? Or a for loop?
+test('should move to right page with feedback resp suggestion link', async ({page}) => {
+  await clickOnLinkAndCheckIfTabOpensWithCorrectURL(page, 'feedbackLinkStarGithub', /.*\/github.com\/citrusframework\/citrus-simulator/);
+  await clickOnLinkAndCheckIfTabOpensWithCorrectURL(page, 'feedbackAndSuggestionLink', /.*\/github.com*/); //could be the login site ore the new-issue site: should one test the two options?
+
+})
+
+test('should move to scenario-results page with right search field params after click on detail buttons', async ({page}) => {
+  for (const element of scenarioSummariesLinkFilterTriples) {
+    await page.getByTestId(element.testName).click();
+    await expect(page).toHaveURL(element.link);
+    await expect(page.getByTestId('scenarioExecutionStatusInSelect')).toHaveValue(element.filterText);
+  }
+})
+
+test('should have updated total, successful, failed tabs after refresh button clicked positive test', async ({page}) => {
+  await checkIfSummaryTabsAreDisplayingRightNumbersFunction(page, nbOfTotalTests, nbOfSuccessfulTests, nbOfFailedTests);
+
+  nbOfFailedTests -= 10;
+  nbOfSuccessfulTests += 10; //the total stays the same
+  await mockBackendResponse(page, '**/api/test-results/count-by-status', {
+    "successful": nbOfSuccessfulTests,
+    "failed": nbOfFailedTests,
+    "total": nbOfTotalTests
+  });
+  await page.getByTestId('refreshListButton').click();
+
+  await checkIfSummaryTabsAreDisplayingRightNumbersFunction(page, nbOfTotalTests, nbOfSuccessfulTests, nbOfFailedTests);
+})
+
+test('should have updated total, successful, failed tabs after refresh button clicked negative test with false total', async ({page}) => {
+  await checkIfSummaryTabsAreDisplayingRightNumbersFunction(page, nbOfTotalTests, nbOfSuccessfulTests, nbOfFailedTests);
+  nbOfFailedTests -= 10; // so the total will be wrong!
+  const newCorrectTotal: number = nbOfTotalTests - 10;
+  await mockBackendResponse(page, '**/api/test-results/count-by-status', {
+    "successful": nbOfSuccessfulTests,
+    "failed": nbOfFailedTests,
+    "total": nbOfTotalTests
+  })
+
+  await page.getByTestId('refreshListButton').click();
+
+  await checkIfSummaryTabsAreDisplayingRightNumbersFunction(page, newCorrectTotal, nbOfSuccessfulTests, nbOfFailedTests);
+})
+
+test('should have same total, successful, failed tabs after cancel deletion via close-Button and cancel-Button', async ({page}) => {
+  await checkIfSummaryTabsAreDisplayingRightNumbersFunction(page, nbOfTotalTests, nbOfSuccessfulTests, nbOfFailedTests);
+  const closeButtons = ['testResultDeleteDialogCloseButton', 'testResultDeleteDialogCancelButton'];
+
+  for (const button of closeButtons) {
+    await page.getByTestId('resetButton').click();
+    await expect(page.getByTestId('testResultDeleteDialogHeading')).toBeVisible();
+    await page.getByTestId(button).click();
```

Comment:
should the test assert there that the api was not called or is that already covered in unit tests normally?

Reply:
api muss geprüft sein weil html-close funktion